internal/mode/static/telemetry/collector.go

```diff
@@ -45,13 +46,63 @@ type ProjectMetadata struct {
 	Version string
 }
 
+// Option 1: list off each Flag and its value
+//type DeploymentFlagOptions struct {
+//	GatewayClass string
+//	GatewayCtlrName string
+//	Gateway                string
+//	Config                 string
+//	Service                string
+//	UpdateGCStatus         bool
+//	MetricsDisable         bool
+//	MetricsSecure          bool
+//	MetricsPort            string
+//	HealthDisable          bool
+//	HealthPort             string
+//	LeaderElectionDisable  bool
+//	LeaderElectionLockName string
+//	Plus                   bool
+//}
+
+// Option 2 doesn't work with exporter, can't handle slices of structs.
+//type DeploymentFlagOptions struct {
+//	NonBooleanFlags []NonBooleanFlag
+//	BooleanFlags    []BooleanFlag
+//}
+//
+//type BooleanFlag struct {
+//	Name  string
+//	Value bool
+//}
+//type NonBooleanFlag struct {
+//	Name  string
+//	Value string
+//}
+
+// Option 2.1: separate Boolean and Non-boolean flags
+//type DeploymentFlagOptions struct {
+//	NonBooleanFlagKeys   []string
+//	NonBooleanFlagValues []string
+//
+//	BooleanFlagKeys   []string
+//	BooleanFlagValues []bool
+//}
+
+// Option 3: Don't separate the boolean and non-boolean flags but simply keep the value of the boolean flag
```

Comment:
Option 3 seems fine to me. I am not sure if it will matter on the XCDF side if we have booleans stored as strings, though. 

What about storing a key-value pair as a string using a `:` as the delimiter: `"plus:true"`?

Reply:
> Option 3 seems fine to me. I am not sure if it will matter on the XCDF side if we have booleans stored as strings, though.

we can use "enabled" and "disabled" instead of "true" and "false" :) 

> What about storing a key-value pair as a string using a : as the delimiter: "plus:true"?

This might make querying the data harder. For example, what flags are most commonly enabled? what flags are most commonly set to non-default values?

Reply:
Yes, these are still useful, primarily for false vs "default or user-specified." I can see a few questions we can answer with this data:

What flags do our users tend to use?
How many of our users are using nginx+?
Are our users disabling health or metrics (likely for a security reason)?

Totally fine if there are no default values - but it does make me think we might want to include a simple "set" or "true" in which case instead of "user-specified" to denote the difference.